src/pages/index.page.jsx

```diff
@@ -45,32 +46,32 @@ export default function Home() {
               <ComponentCard
                 name="Accordion"
                 slug="/accordion"
-                imageOpen="/images/components/accordion_open.svg"
-                imageClosed="/images/components/accordion_closed.svg"
+                imageOpen={details.Accordion.defaultImage}
+                imageClosed={details.Accordion.hoverImage}
```

Comment:
Out of scope, but we should probably change the `imageOpen` and `imageClosed` props to `defaultImage` and `hoverImage`. I'll add a card for it!